### ISSUE TITLE
Symfony Translation Component for Modern Module Controllers

### ISSUE THREAD

**Issue by TechhDan** [PrestaShop/docs#303]
The [Module Translations] documentation does not yet have anything on Modern Controllers. The Core development [section] states the Symfony Translation component is only available for Native and Core modules.

```twig
<!-- can we start doing something like this? -->
{{ 'Dark Theme'|trans({}, 'Modules.ExampleModuleMailtheme') }}
```
Is there a way we can go about translating in our Twig templates or should we wait until this gets implemented later on?

[Module Translations]: https://devdocs.prestashop.com/1.7/modules/creation/module-translation/
[section]: https://devdocs.prestashop.com/1.7/development/internationalization/translation/

**Comment by kpodemski** [comment]
I'm also interested in answer, how to translate .twig templates?

cc @Quetzacoalt91 @PierreRambaud @eternoendless

**Comment by eternoendless** [comment]
I've been [working on it lately](https://github.com/PrestaShop/docs/issues/284). There should have a PR ready soon.

The answer is YES you can starting on 1.7.6, BUT there's a known bug where translations from DB aren't working on 1.7.6.0 ([being fixed for 1.7.6.1](https://github.com/PrestaShop/PrestaShop/pull/14845)), and support is incomplete because you can't export them to a file yet (but you can use legacy translation files if you know how to make them).

**Comment by eternoendless** [comment]
Duplicate of #284
